<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExQStandardItemModel</class>
 <widget class="QMainWindow" name="ExQStandardItemModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExQStandardItemModel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Table View:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>PlainTextEdit:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpen"/>
   <addaction name="actSave"/>
   <addaction name="actModelData"/>
   <addaction name="separator"/>
   <addaction name="actAppend"/>
   <addaction name="actInsert"/>
   <addaction name="actDelete"/>
   <addaction name="separator"/>
   <addaction name="actAlignLeft"/>
   <addaction name="actAlignCenter"/>
   <addaction name="actAlingRight"/>
   <addaction name="actBold"/>
   <addaction name="separator"/>
   <addaction name="actExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image011.jpg</normaloff>:/images/Image011.jpg</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件夹</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image001.png</normaloff>:/images/Image001.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actAppend">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image012.png</normaloff>:/images/Image012.png</iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
   <property name="toolTip">
    <string>添加一行</string>
   </property>
  </action>
  <action name="actInsert">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image008.jpg</normaloff>:/images/Image008.jpg</iconset>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
   <property name="toolTip">
    <string>插入一行</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image009.jpg</normaloff>:/images/Image009.jpg</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除一行</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image007.jpg</normaloff>:/images/Image007.jpg</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actModelData">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image010.jpg</normaloff>:/images/Image010.jpg</iconset>
   </property>
   <property name="text">
    <string>预览</string>
   </property>
   <property name="toolTip">
    <string>预览数据模型到文本框</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actAlignLeft">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image004.png</normaloff>:/images/Image004.png</iconset>
   </property>
   <property name="text">
    <string>居左</string>
   </property>
   <property name="toolTip">
    <string>文字左对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actAlignCenter">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image006.png</normaloff>:/images/Image006.png</iconset>
   </property>
   <property name="text">
    <string>居中</string>
   </property>
   <property name="toolTip">
    <string>文本居中</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actAlingRight">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image005.png</normaloff>:/images/Image005.png</iconset>
   </property>
   <property name="text">
    <string>居右</string>
   </property>
   <property name="toolTip">
    <string>文本右对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="actBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image003.png</normaloff>:/images/Image003.png</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>文字设置粗体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
